### PR TITLE
build: no longer require updating multiple package.json files for bazel

### PR DESCRIPTION
**Pull request by DevVersion**
* Replaces second "package.json" file with a small workspace that just aliases to the actual dependencies provided in the "@matdeps" workspace. This speeds up the analysis phase and also makes debugging easier.
* Also proxies the "ng_package" so that it always includes the proper `README.md` file. Also this means that we can always replace the version placeholders. Similar to `angular/angular`.

Similar to https://github.com/angular/angular/commit/68074df0a23f12b11bd480a1911d7403d2b31530. Credits to @gregmagolan for this.

We just need a bit more work because we don't overwrite the default `compiler`, `xi18n` and `packager` targets to the actual source files (this is special to Angular because those contain the sources)

### CODE REVIEW

File: src/BUILD.bazel [hunk 1]
```diff
@@ -5,6 +5,7 @@ load("//tools:defaults.bzl", "ts_library")
 exports_files([
   "bazel-tsconfig-build.json",
   "bazel-tsconfig-test.json",
+  "README.md",
```
Comment (jelbourn) [inline src/BUILD.bazel]: Not for this PR, but we should create separate npm README files for each of our packages

File: tools/npm-workspace/@angular/bazel/bin/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package(default_visibility = ["//visibility:public"])
+
+# Workaround that ensures that our our build targets using the Angular Bazel rules
```
Comment (jelbourn) [inline tools/npm-workspace/@angular/bazel/bin/BUILD.bazel]: "our our" (here and in the other alias file)

File: src/BUILD.bazel (hunk 1 above)
Comment (DevVersion) [reply src/BUILD.bazel]: Agreed. Should these be pretty much similar to the current one but just have the "correct" package name title and refer to the proper URL for the sources?

Comment (jelbourn) [reply src/BUILD.bazel]: I created an issue and assigned it to myself for this.

**Pull request merged**